browser/src/canvas/sections/CommentSection.ts

```diff
@@ -320,6 +321,14 @@ export class Comment extends CanvasSectionObject {
 		this.sectionProperties.menu.annotation = this;
 	}
 
+	private createReplyHint (): void {
+		this.sectionProperties.replyHint = L.DomUtil.create('p', '', this.sectionProperties.nodeModify);
+		var small = document.createElement('small');
+		small.classList.add('cool-font');
+		small.innerText = 'Press Ctrl + Enter to post';
```

Comment:
please make the text possible to be localized (translated) by placing the text in between `_()`. See the example above: `_('Edit comment')` (so it gets localized https://github.com/CollaboraOnline/online/blob/1738a8e59a5ef6d06ae95b1280483f568445d860/browser/js/global.js#L1691)